dom0/sd-clean-all.sls

```diff
@@ -117,3 +82,19 @@ disable-systemd-units:
       # Even with "runas", "systemctl --user" from root will fail unless we
       # tell it explicitly how to connect to the user systemd.
       - XDG_RUNTIME_DIR: /run/user/{{ gui_user_id }}
+
+# Reset desktop icon size to its original value
+dom0-reset-icon-size-xfce:
+  cmd.script:
+    - name: /usr/bin/securedrop/update-xfce-settings
+    - args: reset-icon-size
+    - runas: {{ gui_user }}
```

Comment:
(Eventually I want to clean this up and put it in the %preun of the rpm, but in the interests of time, it's still using Salt for the cleanup.)